### PR TITLE
Mariposa

### PR DESCRIPTION
**Pull request by nathaniellers**
Hacktoberfest!

### CODE REVIEW

File: src/recipes/mariposa-drink.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            "quantity": "1",
+            "measure": "tablespoon",
+            "ingredient": "orange"
+        }
+    ],
+    "directions": [
+        "Shake all ingredients together with ice",
+        "strain into a cocktail glass"
+    ],
+    "image": "mariposa,jpg",
```
Comment (alfg) [inline src/recipes/mariposa-drink.json:31]:
```suggestion
    "image": "mariposa.jpg",
```

**Review by alfg — CHANGES_REQUESTED** [review]
Added suggested to correct image path. Thanks!

File: src/recipes/mariposa-drink.json (hunk 1 above)
Comment (nathaniellers) [reply src/recipes/mariposa-drink.json:31]: oh.. sorry. Thanks!

File: src/recipes/dalogna-coffee.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            "ingredient": "milk"
+        }
+    ],
+    "directions": [
+        "In a bowl, combine coffee granules, sugar, and water.",
+        "Using an electric mixer, beat for about 5 to 7 minutes or until stiff peaks form.",
+        "Fill two serving glasses with ice cubes up to 3/4 full. Pour milk enough to cover the ice.",
+        "Spoon coffee foam on top of milk and stir. Serve with a straw."
+    ],
+    "image": "mariposa.jpg",
```
Comment (alfg) [inline src/recipes/dalogna-coffee.json:33]:
```suggestion
    "image": "dalgona-coffee.jpg",
```

File: src/recipes/dalogna-coffee.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Dalogna Coffee",
```
Comment (alfg) [inline src/recipes/dalogna-coffee.json:2]: This should be `Dalgona Coffee`, right?

**Review by alfg — CHANGES_REQUESTED** [review]
Two more fixes suggested. Also, can you keep the image size below 200KB, please? Thanks!

**Comment by alfg** [comment]
Closing. As dalgona coffee conflicts with existing recipe. Feel free to re-open another with the fixed Mariposa recipe. Thanks.

**Pull request closed without merging**